__tests__/app.test.js

```
@@ -133,6 +133,22 @@ describe("app.js", () => {
             });
           });
       });
+      test("GET:200: should return with a list of articles filtered by specified topic when given a valid topic", () => {
+        return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles.length).toBeGreaterThanOrEqual(12)
+        })
+      })
+      test("GET:200: should return with an empty array when passed a topic that doesn't exist", () => {
+        return request(app)
+        .get("/api/articles?topic=topicthatnoexist")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles).toEqual([])
+        })
+      })
```

This should actually throw an error as topicthatnoexist is not a valid topic that we can sort by, but we should have a test for a topic that doesnt have an associated articles with it which should give back an empty array